src/ContainerCommitModal.jsx

```diff
@@ -112,113 +93,73 @@ class ContainerCommitModal extends React.Component {
                 </FormGroup>
 
                 <FormGroup fieldId="commit-dialog-author" label={_("Author")}>
                     <TextInput id="commit-dialog-author"
+                               placeholder={_("Example, Your Name <yourname@exmaple.com>")}
                                value={this.state.author}
                                onChange={value => this.handleInputChange("author", value)} />
                 </FormGroup>
 
-                <FormGroup fieldId="commit-dialog-message" label={_("Message")}>
-                    <TextInput id="commit-dialog-message"
-                               value={this.state.message}
-                               onChange={value => this.handleInputChange("message", value)} />
-                </FormGroup>
-
                 <FormGroup fieldId="commit-dialog-command" label={_("Command")}>
                     <TextInput id="commit-dialog-command"
                                value={this.state.command}
                                onChange={value => this.handleInputChange("command", value)} />
                 </FormGroup>
 
-                <FormGroup fieldId="commit-dialog-pause">
+                <FormGroup fieldId="commit-dialog-pause" label={_("Options")} isStack hasNoPaddingTop>
                     <Checkbox id="commit-dialog-pause"
                               isChecked={this.state.pause}
                               onChange={value => this.handleInputChange("pause", value)}
-                              label={_("Pause the container")} />
-                </FormGroup>
-
-                <FormGroup fieldId="commit-dialog-setonbuild" isStack>
-                    <Checkbox id="commit-dialog-setonbuild" isDisabled={this.state.onbuildDisabled}
-                              isChecked={this.state.setonbuild}
-                              onChange={value => this.handleInputChange("setonbuild", value)}
-                              label={_("Set container on build variables")} />
-                    {this.state.setonbuild && onbuilds}
+                              label={_("Pause container when creating image")} />
+                    <Checkbox id="commit-dialog-docker"
+                              isChecked={this.state.useDocker}
+                              onChange={value => this.handleInputChange("useDocker", value)}
+                              description={_("Docker format is useful when sharing the image with Docker or Mody Engine")}
+                              label={_("Use legacy Docker format")} />
                 </FormGroup>
             </Form>;
 
         return (
             <Modal isOpen
                    showClose={false}
                    position="top" variant="medium"
-                   title={_("Commit image")}
+                   title={_("Commit container")}
+                   description={cockpit.format(_("Create a new image based on the current state of the $0 container."), this.props.container.Names)}
```

Comment:
Are we able to wrap the container name with a `<strong>` element? Or do descriptions only support text-only?

Reply:
Marius has a clever [fmt_to_fragments()](https://github.com/cockpit-project/cockpit/blob/main/pkg/storaged/utilsx.jsx) helper which can help with translating strings with markup. We could move that to pkg/lib for sure.

Reply:
Done